arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -174,6 +178,10 @@
                     <string>MobileMapSearchRoute</string>
                     <key>descriptionText</key>
                     <string>This sample shows how to search and route using data in a map package</string>
+                    <key>dependency</key>
+                    <array>
+                        <string>MobileMapPackages</string>
```

Comment:
How do these strings get mapped to the actual files/folders that need to be downloaded?